src/toil_scripts/lib/jobs.py

```diff
@@ -0,0 +1,22 @@
+from toil_scripts.lib import partitions
+
+
+def sample_batcher_job(job, func, samples, *args):
```

Comment:
Also mention for which inputs sizes it would be appropriate or necessary to use this function.